web/html/src/manager/virtualization/guests/list/guests-list.js

```diff
@@ -372,24 +372,27 @@ class GuestsList extends React.Component<Props, State> {
                             const state = row.stateLabel;
                             return (
                               <div className="btn-group">
-                                {state !== 'running'
+                                {state !== 'running' && row.name !== 'Domain-0'
                                  && (
                                    <AsyncButton
                                      defaultType="btn-default btn-sm"
                                      title={t(state === 'paused' ? 'Resume' : 'Start')}
                                      icon="fa-play"
                                      action={() => onAction('start', [row.uuid], {})}
                                    />) }
-                                {state === 'running' && this.createModalButton('suspend', modalsData, row) }
-                                {state !== 'stopped' && this.createModalButton('shutdown', modalsData, row) }
+                                {state === 'running' && row.name !== 'Domain-0'
```

Comment:
Why `Domain-0` has special treatment? What does it represent?

Reply:
Please add a comment with this explanation otherwise I think it's not really clear to the developer.